migrations/sql/V2021.07.09_1359__WorkflowRunReports.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE orderlyweb_workflow_run_reports
+(
+    id              INTEGER PRIMARY KEY,
+    workflow_key    TEXT      NOT NULL,
+    report_key      TEXT      NOT NULL UNIQUE,
+    name            TEXT      NOT NULL,
```

Comment:
For consistency with `orderlyweb_report_run` it would be better for these columns to be named `key` and `report`.

Later on we can consider adding `status`, `logs` and `report_version` in order to persist information from orderly.server's status endpoint.

Reply:
There are two keys and adding prefix makes life easier for identification. workflow_key and report_key.

Reply:
`workflow_key` makes a lot of sense. But consistency is important, so `report_key` and `name` should be aligned with `orderlyweb_report_run` and named `key` and `report` respectively. If we don't it will also make it harder to unify the `orderlyweb_report_run` and `orderlyweb_workflow_run_reports` tables at some point in the future.